server/src/ClientToDBAPI/SynchronizationDataRoute/mainRoute.ts

```diff
@@ -1,23 +1,38 @@
 import axios from 'axios';
 import { Router, Request, Response } from 'express';
 
-import City from '../../Models/Address/City';
 import { Severity } from '../../Models/Logger/types';
-import {ADD_CITY_TEMP, ADD_CITIES_TEMP} from '../../DBService/DataSynchronization/mutation';
 import { errorStatusCode, graphqlRequest } from '../../GraphqlHTTPRequest';
+import { GET_LAST_STREET_ID } from '../../DBService/DataSynchronization/Query';
+import { ADD_CITIES, ADD_STREETS} from '../../DBService/DataSynchronization/Mutation';
 import logger, { invalidAPIResponseLog, launchingAPIRequestLog, invalidDBResponseLog, launchingDBRequestLog, validDBResponseLog  } from '../../Logger/Logger';
 
+
 const synchronizationRoute = Router();
+
+// const mohCitiesApiUrl = process.env.CITIES_URL;
+// const mohStreetsApiUrl = process.env.STREETS_URL;
 const mohCitiesApiUrl = 'https://mohservicesapitest.health.gov.il/api/Lists/edm/israelcities';
 const mohStreetsApiUrl = 'https://mohservicesapitest.health.gov.il/api/Lists/edm/israelstreets';
 
```

Comment:
Do you want to write the URL hardcoded? Why you don't use the environment variable?

Reply:
👍